pc_autobackup.py

```diff
@@ -239,42 +247,34 @@ def main():
     update_config = True
 
   if update_config:
-    with open(options.config_file, 'wb') as config_file:
+    with open(common.CONFIG_FILE, 'wb') as config_file:
       config.write(config_file)
 
   if options.create_camera_config:
-    UpdateCameraConfig(options.create_camera_config, create_desc_file=True,
-                       config_file=options.config_file)
+    UpdateCameraConfig(options.create_camera_config, create_desc_file=True)
     sys.exit(0)
 
   if options.import_camera_config:
-    ImportCameraConfig(options.import_camera_config,
-                       config_file=options.config_file)
+    ImportCameraConfig(options.import_camera_config)
     sys.exit(0)
 
   if options.update_camera_config:
-    UpdateCameraConfig(options.update_camera_config,
-                       config_file=options.config_file)
+    UpdateCameraConfig(options.update_camera_config)
     sys.exit(0)
 
-  if config.has_option('AUTOBACKUP', 'default_interface'):
-    interface = config.get('AUTOBACKUP', 'default_interface')
-    logger.info('PCAutoBackup started on %s', interface)
-  else:
-    interface = ""
-    logger.info('PCAutoBackup started on all interfaces')
-
+  logger.info('PCAutoBackup started on %s', config.get('AUTOBACKUP',
+                                                       'default_interface'))
   logger.info('Server name: %s', config.get('AUTOBACKUP', 'server_name'))
 
   if options.debug:
-    GetSystemInfo(config_file=options.config_file)
+    GetSystemInfo()
 
-  reactor.listenMulticast(1900, ssdp.SSDPServer(options.config_file), listenMultiple=True)
+  reactor.listenMulticast(1900, ssdp.SSDPServer(), listenMultiple=True)
   logger.info('SSDPServer started')
 
-  resource = mediaserver.MediaServer(options.config_file)
+  resource = mediaserver.MediaServer()
   factory = Site(resource)
-  reactor.listenTCP(52235, factory, interface=interface)
+  reactor.listenTCP(52235, factory)
```

Comment:
This doesn't do what you want it to. By default twisted will listen on all interfaces 0.0.0.0 adding interface= you are able to tell twisted which interface to bind to. Again, you are changing the behavior of existing clients with these changes. Also, you added a bunch of code that is now completely ignored by this change and completely contradicts with the logging statement on line 265.